src/index.md

```diff
@@ -15,15 +15,17 @@ Get started easily, get more done, and never play twenty questions with <code>ku
 <a href="{{site.docsurl}}install.html" class="brandButton u-marginBottomUnit u-marginRightUnit">
   {% include brandButtonBg.html %}
   <div class="buttonLabel brandButton-text">
-    Try Tilt Today
+    Try Tilt Now
   </div>
 </a>
 
-<h3 class="ctaLink u-marginBottomUnit">
-  <a href="https://github.com/windmilleng/tilt">
-    &hairsp;&gt; Check us out on GitHub&hairsp;
-  </a>
-</h3>
+<div class="flexblock u-marginTop1_25 u-marginBottom1_5">
+<a href="https://cloud.tilt.dev/team/new" class="brandButton u-marginBottomUnit u-marginRightUnit">
+  {% include brandButtonBg.html %}
+  <div class="buttonLabel brandButton-text">
+    Tilt for Work
```

Comment:
The two options "Try Tilt Now" vs "Tilt for Work" as identical buttons are confusingly similar. A good call-to-action make it super easy to know what to do next. Here, I'm forced to think about the distinction between these two nearly identical options. (Aka, violating the [Don't Make Me Think](https://www.amazon.com/Dont-Make-Me-Think-Usability/dp/0321344758) commandment of good usability.)

### My Proposal
Change the primary button to "Get Tilt for your Team" (or similar). And leep the GitHub and Docs links as a secondary CTA. Like this:

<img width="1213" alt="Screen Shot 2020-04-20 at 6 17 05 PM" src="https://user-images.githubusercontent.com/20349/79805011-37be2380-8333-11ea-9237-c24672370ece.png">